docs/installation.md

```diff
@@ -69,6 +69,10 @@ for the Actor and a few configuration files.
 
 ### Create a New Actor from Existing Project
 
+:::note Don't use this after `apify create`
```

Comment:
Maybe something like that for the note title

```suggestion
:::note Automatic Actor directory initialization
```

Reply:
I like it, but do you think it will stop them? 😄 Maybe we could use your text, but wrap it in a warning instead of a note?

Reply:
Wouldn't `tip` be better? Or do we want to emphasize to user that this might have adverse effects and we strongly recommend against it? Or we just want to mention that it is not necessary to do it after `apify create`?

Reply:
That's a good question! 😅 I tested it and it seems the CLI recovers gracefully in that situation, so we can make it a tip 👍